testing-utils/src/lib/utils/minimal-config.mock.ts

```diff
@@ -43,7 +44,7 @@ export const MINIMAL_PLUGIN_CONFIG_MOCK: PluginConfig = {
     },
   ],
   runner: MINIMAL_RUNNER_FUNCTION_MOCK,
-};
+} satisfies PluginConfig;
```

Comment:
Why use `satisfies` instead of a type annotation?

Reply:
Good point, fixed!